packages/caip/src/adapters/junopay/index.ts

```diff
@@ -0,0 +1,80 @@
+import entries from 'lodash/entries'
+import invert from 'lodash/invert'
+import toLower from 'lodash/toLower'
+
+import { AssetId, fromAssetId } from '../../assetId/assetId'
+import { ChainId } from '../../chainId/chainId'
+import { avalancheAssetId, btcAssetId, cosmosAssetId, ethAssetId } from './../../constants'
+
+const assetIdToJunopayTickerMap: Record<AssetId, string> = {
```

Comment:
Can we comment this, adding a link to the docs where we can double check these tickers? Looking at https://docs.juno.finance/developers/reference/onjuno-pay/onjuno-pay-integration-guide this is correct, but in https://api.juno.finance/crypto-wallet-partners?partner_key=live_secretAPIKey, the `from_currency` and `to_currency` are actually all caps.

I'd probably honor the latter and uppercase the ticker here, since we then lowercase it in the queryparams creation in https://github.com/shapeshift/web/pull/2125/files#diff-792b57002da5047c7246b59ba3e52bd7989c3af6b91201c32171ca72a1903243R49

Reply:
i thought the same looking at this too in conjunction with web. i'm half and half on this. it's silly they mix uppercase and lowercase but we gotta deal.

the only thing that makes me inclined to keep it lowercase here, is it's a longer tail of lowercase tokens. either way we've gotta convert one way to the other in web.

do you feel strongly either way, or should we just ship? i'm going to clean up the respective web PR now regardless.

Reply:
I'm fine with either, would feel cleaner to use the tickers from the supported assets API, but at the end of the day until this is properly documented on Juno side, doesn't matter which one we use